dockstore-integration-testing/src/test/java/io/dockstore/webservice/SearchResourceIT.java

```diff
@@ -96,6 +97,8 @@ public void testSearchOperations() throws ApiException {
         waitForRefresh(1500);
         // after publication index should include workflow
         String s = extendedGa4GhApi.toolsIndexSearch(exampleESQuery);
+        assertTrue(s.contains("\"aliases\":{}"));
```

Comment:
very close, let's add an actual alias on workflow before `waitForRefresh` so it shows up here